### PR TITLE
New Discover Welcome Banner

### PR DESCRIPTION
**Pull request by leandroalonso**
Part of #14399

This PRs adds the welcome banner the first time the user sees the Discover section.

### iPhone

| Light  | Dark |
| ------------- | ------------- |
| Simulator Screen Shot - iPhone 11 - 2020-07-14 at 09 49 04 | Simulator Screen Shot - iPhone 11 - 2020-07-14 at 09 48 54 |

### iPad

| Light  | Dark |
| ------------- | ------------- |
| Simulator Screen Shot - iPad Pro (9 7-inch) - 2020-07-14 at 09 53 01 | Simulator Screen Shot - iPad Pro (9 7-inch) - 2020-07-14 at 09 51 51 |

### To test

**Make sure that you have overridden the Reader Improvements Phase 2 Feature Flag (Me -> App Settings -> Debug -> "Reader Improvements Phase 2" should be ON)**

1. Go to the Discover section
2. Check that the welcome banner appear at the top
3. Change to another section
4. Go back, check that the welcome banner doesn't appear

PR submission checklist:

- [x] I have considered adding unit tests where possible.
- [x] I have considered adding accessibility improvements for my changes.
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: WordPress/Classes/ViewRelated/Reader/ReaderWelcomeBanner.swift [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+    override func awakeFromNib() {
+        super.awakeFromNib()
+        applyStyles()
+        configureWelcomeLabel()
+    }
+
+    /// Present the Welcome banner just one time
+    class func presentIfNeeded(in tableView: UITableView) {
+        guard !UserDefaults.standard.bool(forKey: ReaderWelcomeBanner.bannerPresentedKey) else {
```
Comment (emilylaguna) [inline WordPress/Classes/ViewRelated/Reader/ReaderWelcomeBanner.swift:16]: Should we make a test to make sure this works as intended?

**Comment by leandroalonso** [comment]
@emilylaguna Done!

### Portrait

| iPad  | iPhone |
| ------------- | ------------- |
| Simulator Screen Shot - iPad Pro (9 7-inch) - 2020-07-15 at 13 15 46 | Simulator Screen Shot - iPhone 11 - 2020-07-15 at 13 14 29 |

### Landscape
| iPad  | iPhone |
| ------------- | ------------- |
| Simulator Screen Shot - iPad Pro (9 7-inch) - 2020-07-15 at 13 15 49 | Simulator Screen Shot - iPhone 11 - 2020-07-15 at 13 14 33 |

**Comment by leandroalonso** [comment]
@emilylaguna now is ready for another review!

**Comment by leandroalonso** [comment]
@mbshakti I added more dots for iPad:

### Landscape

Simulator Screen Shot - iPad Pro (9 7-inch) - 2020-07-15 at 17 40 07

### Portrait

Simulator Screen Shot - iPad Pro (9 7-inch) - 2020-07-15 at 17 40 00

**Comment by leandroalonso** [comment]
@mbshakti what do you think about @emilylaguna's suggestion?

Then more dots will appear whenever we have more space.

**Comment by mbshakti** [comment]
@leandroalonso  Makes total sense. Looks great to me.

**Pull request merged**